rpm/SPECS/docker-ce-cli.spec

```diff
@@ -109,7 +88,6 @@ done
 %files
 %doc build-docs/LICENSE build-docs/MAINTAINERS build-docs/NOTICE build-docs/README.md
 %{_bindir}/docker
-%{_libexecdir}/docker/cli-plugins/*
```

Comment:
needs to remove that line to fix: https://ci-next.docker.com/public/blue/organizations/jenkins/docker-ce-packaging/detail/PR-654/2/pipeline#step-893-log-356

```
PASS: cli version OK

Processing files: docker-ce-cli-0.0.0.20220325091330.87b8e57-0.el7.aarch64

error: File not found by glob: /root/rpmbuild/BUILDROOT/docker-ce-cli-0.0.0.20220325091330.87b8e57-0.el7.aarch64/usr/libexec/docker/cli-plugins/*

Executing(%doc): /bin/sh -e /var/tmp/rpm-tmp.hH1z6a
```

we don't ship plugins inside docker-ce-cli pkg anymore.